Tribler/Test/test_as_server.py

```diff
@@ -187,7 +188,7 @@ def tearDown(self):
         """ unittest test tear down code """
         if self.session is not None:
             self._shutdown_session(self.session)
-            Session.del_instance()
+        Session.del_instance()
```

Comment:
This is intentional, if we created a session, call del_instance. If not (self.session = None) don't.